config/README.md

```diff
@@ -84,8 +84,8 @@ Else if you just run the script by just:
 ```
 htcanalyze
 ```
-This will search for the config_file with the priorities from 1 (high) to 4 (low)
-if NOT --no-config Flag is set:
+Confusing word order, better:
```

Comment:
Don't use comments as a conversational platform. You can describe this in a commit, a PR message, or even as a comment to a review question that someone may ask. Right now it means that if this PR is accepted, this will be merged with master and the comment doesn't do anything to explain the code, just your change, which outside of the PR context is confusing (I'm only commenting on it here, but it happens several times).